comfy/model_management.py

```diff
@@ -650,8 +662,14 @@ def get_free_memory(dev=None, torch_free_too=False):
         dev = get_torch_device()
 
     if hasattr(dev, 'type') and (dev.type == 'cpu' or dev.type == 'mps'):
-        mem_free_total = psutil.virtual_memory().available
-        mem_free_torch = mem_free_total
+        mem_total = get_containerd_memory_limit()
+        if mem_total > 0:
+            mem_used_total = psutil.virtual_memory().used
```

Comment:
In some cases, the value obtained by `psutil.virtual_memory().used` is also incorrect

Reply:
Maybe can read file `/sys/fs/cgroup/memory/memory.usage_in_bytes` to get correct memory used

Reply:
yeah, I fixed it